### DISCUSSION TITLE
how to delete a line

### DISCUSSION THREAD

**Discussion by wgqimut** [helix-editor/helix#7071] [Q&A] [answered]
In "migrating from vim" wiki, I don't see how to delete a line, in vim it's `dd`, can it possible to do it shortly in helix?

**Comment by yuyidegit** [comment, marked as answer]
xd

**Reply by wgqimut** [reply]
Thanks a lot!

**Comment by pkulak** [comment]
Well, it's kinda xd, except if you are on an empty line, then you delete two lines... so really it's Xd.
